Lib/asyncio/locks.py

```diff
@@ -24,22 +24,22 @@ class Lock(_ContextManagerMixin, mixins._LoopBoundMixin):
     """Primitive lock objects.
 
     A primitive lock is a synchronization primitive that is not owned
-    by a particular coroutine when locked.  A primitive lock is in one
+    by a particular task when locked.  A primitive lock is in one
     of two states, 'locked' or 'unlocked'.
 
     It is created in the unlocked state.  It has two basic methods,
     acquire() and release().  When the state is unlocked, acquire()
     changes the state to locked and returns immediately.  When the
     state is locked, acquire() blocks until a call to release() in
-    another coroutine changes it to unlocked, then the acquire() call
+    another task changes it to unlocked, then the acquire() call
     resets it to locked and returns.  The release() method should only
     be called in the locked state; it changes the state to unlocked
     and returns immediately.  If an attempt is made to release an
     unlocked lock, a RuntimeError will be raised.
 
-    When more than one coroutine is blocked in acquire() waiting for
-    the state to turn to unlocked, only one coroutine proceeds when a
-    release() call resets the state to unlocked; first coroutine which
+    When more than one task is blocked in acquire() waiting for
+    the state to turn to unlocked, only one task proceeds when a
+    release() call resets the state to unlocked; first task which
     is blocked in acquire() is being processed.
```

Comment:
I am unsure what the final phrase here is trying to say. Maybe it is hinting at FIFO order. If so, I suggest that we drop it since it is an implementation/policy detail.

Reply:
Surprisingly many people really care about fairness in locks and queues, and would complain loudly if a fair implementation was replaced with a less fair one. Since we believe it's now fair, I think we should document that. The sentence is garbled, here's a suggestion:
```suggestion
    release() call resets the state to unlocked; successive release()
    calls will unblock tasks in FIFO order.
```